org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/edmmMappings/edmmMappings.service.ts

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { backendBaseURL } from '../../../configuration';
+import { Router } from '@angular/router';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { SectionData } from '../../../section/sectionData';
+import { ToscaTypes } from '../../../model/enums';
+
+export enum EdmmType {
+    compute = 'compute',
+    database = 'database',
```

Comment:
Enums upper case?!

Reply:
Yes, but then we have also uppercase in the json file we use to save. I thought it would be better to have EDMM convention in the file, if someone wants to edit it manually.

Reply:
What about overwriting `toString()` and returning the String value you have already defined. AFAIK Jackson calls `toString()`...